src/commands/deployImage/imageSource/containerRegistry/acr/RegistryEnableAdminUserStep.ts

```diff
@@ -30,7 +29,6 @@ export class RegistryEnableAdminUserStep extends AzureWizardPromptStep<IContaine
     }
 
     public shouldPrompt(context: IContainerRegistryImageContext): boolean {
-        // this is only a requirement for ACR
-        return context.registryDomain === acrDomain && !!context.registry && !context.registry.adminUserEnabled;
+        return !!context.registry && !context.registry.adminUserEnabled;
```

Comment:
Is checking for an acrDomain not a requirement anymore? Or do we know for a fact that this step is only called for acrs?

Reply:
My thought was that `context.registry` has a type definition tied to being an ACR already and so we probably don't need to check for the `acrDomain` if the registry is populated 🤔